src/components/studies/participantHooks.ts

```diff
@@ -0,0 +1,90 @@
+import { useUserSessionDataState } from "@helpers/AuthContext"
+import ParticipantUtility, {ParticipantData} from "./participants/participantUtility"
+import { ExtendedError, ExtendedParticipantAccountSummary, ParticipantAccountSummary, ParticipantActivityType, ParticipantEvent } from "@typedefs/types"
+import { useMutation, useQuery, useQueryClient } from "react-query"
+import ParticipantService from "@services/participants.service"
+import EventService from "@services/event.service"
+
+export const PARTICIPANT_KEYS = {
+    all: ['participants'] as const,
+    list: (studyId: string | undefined, currentPage: number = 0, pageSize: number = 0, tab: ParticipantActivityType = 'ACTIVE', searchValue = undefined) =>
+      [...PARTICIPANT_KEYS.all, 'list', studyId, currentPage, pageSize, tab] as const,
+  
+    detail: (studyId: string, userId: string) =>
+      [...PARTICIPANT_KEYS.list(studyId), userId] as const,
+}
+
+export const useParticipants = (
+    studyId: string | undefined, 
+    currentPage: number, 
+    pageSize: number,
+    tab: ParticipantActivityType, 
+    searchValue?: string
+    ) => {
+    const {token} = useUserSessionDataState()
+
+    return useQuery<ParticipantData, ExtendedError>(
+        PARTICIPANT_KEYS.list(studyId, currentPage, pageSize,tab),
+        () => ParticipantUtility.getParticipants(studyId!,currentPage,pageSize,tab, token!),
+        {
+            enabled: !!studyId,
+            retry: false,
+            refetchOnWindowFocus: false,
+        }
+    )
+}
+
+export const useUpdateParticipantInList = () => {
+    const {token} = useUserSessionDataState()
+    const queryClient = useQueryClient()
+
+    const update = async(props: {
```

Comment:
We already have a hook to update the events so this hook should only be updating the fields on participant, Also I don't think we need the page (current/size/tab) information. I think we can invalidate all of the  PARTICIPANT_KEYS.list(props.studyId) not just for the particular page.